### PR TITLE
(PC-13300) fix(PhoneValidation): Fix phone validation modal

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-13300

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: | 
|  image   |    image     |                |      Capture d’écran 2022-02-17 à 15 38 50      |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/auth/signup/PhoneValidation/SetPhoneValidationCode.tsx [hunk 1]
```diff
@@ -271,8 +272,7 @@ export const SetPhoneValidationCode = memo(function SetPhoneValidationCodeCompon
             <Typo.Body>{t`Tu n'as pas reçu le sms\u00a0?`}</Typo.Body>
             {/* force button to wrap on small screen, otherwise timer will "unwrap" when timer is under 10 seconds */}
             {appContentWidth <= 320 ? <Break /> : <Spacer.Row numberOfSpaces={1} />}
-            <ButtonTertiary
-              inlineHeight="100%"
```
Comment (kopax-polyconseil) [inline src/features/auth/signup/PhoneValidation/SetPhoneValidationCode.tsx:275]: Logiquement, ce style est appliqué directement sur le TouchableOpacity via: https://github.com/pass-culture/pass-culture-app-native/blob/master/src/ui/components/buttons/AppButton/styleUtils.ts#L56

Pourquoi cette ligne n'est pas executé ?

Comment (kopax-polyconseil) [reply src/features/auth/signup/PhoneValidation/SetPhoneValidationCode.tsx:275]: Aussi, c'est une customisation graphique, utilisé que par un seul composant, je me demande si on en devrait pas l'interdire.

Comment (aliraiki) [reply src/features/auth/signup/PhoneValidation/SetPhoneValidationCode.tsx:275]: Pour l'instant j'enlève `inlineHeight` ici, ça n'a pas l'air d'avoir d'effet visuel sur Web/iOS/Android

**Review by Lucasbeneston — APPROVED** [review]
Top !! 🚀

**Pull request merged**